python/Dockerfile.tpl

```diff
@@ -3,14 +3,12 @@ FROM python:{{PYTHON_VERSION}}-alpine3.7
 ARG CI_HELPER_VERSION
 
 RUN echo "Starting ..." && \
-    apk add --update -q --no-cache bash curl git make openssh-client rsync tzdata && \
+    apk add --update -q --no-cache bash curl git make openssh-client rsync tzdata build-base libffi-dev openldap-dev && \
```

Comment:
Can you sort the packages please and update the CHANGELOG.md?

Reply:
Changelog updated and packages sorted !